doc/1/getting-started/kotlin/index.md

```diff
@@ -28,6 +28,32 @@ Having trouble? Get in touch with us on [Discord](http://join.discord.kuzzle.io)
 
 You can find the SDK JARs directly on [bintray](https://bintray.com/kuzzle/maven/sdk-jvm). Download and add them to your classpath.
 
+::: warning
+If you have duplicates dependencies issues, you should either add those lines in your application `gradle.build file`
+
+```groovy
+configurations {
+     cleanedAnnotations
+     compile.exclude group: 'org.jetbrains'
+     compile.exclude group: 'org.jetbrains.kotlin'
+ }
+```
+
+Or you can use our jar dependencies less, and add those dependencies in your application:
+
+```groovy
+    implementation("io.ktor:ktor-client-websockets:1.5.2")
+    implementation("io.ktor:ktor-client-okhttp:1.5.2")
+    implementation("io.ktor:ktor-client-cio:1.5.2")
+    implementation("io.ktor:ktor-client-json:1.5.2")
+    implementation("io.ktor:ktor-client-gson:1.5.2")
+    implementation("io.ktor:ktor-client-serialization:1.5.2")
+    implementation("com.google.code.gson:gson:2.8.5")
+```
+
+:::
```

Comment:
::: warning
If you are using the `sdk-jvm-X.Y.Z-with-dependencies.jar` in a Kotlin Android Studio project, consider adding the following lines to your `gradle.build file`

```groovy
configurations {
     cleanedAnnotations
     compile.exclude group: 'org.jetbrains'
     compile.exclude group: 'org.jetbrains.kotlin'
 }
``` 
:::

::: warning

If you are having troubles with duplicate dependencies in your project, consider using the  `sdk-jvm-X.Y.Z-without-dependencies.jar` and add the following dependencies yourself:

```groovy
    implementation("io.ktor:ktor-client-websockets:1.5.2")
    implementation("io.ktor:ktor-client-okhttp:1.5.2")
    implementation("io.ktor:ktor-client-cio:1.5.2")
    implementation("io.ktor:ktor-client-json:1.5.2")
    implementation("io.ktor:ktor-client-gson:1.5.2")
    implementation("io.ktor:ktor-client-serialization:1.5.2")
    implementation("com.google.code.gson:gson:2.8.5")
``` 
:::